### PR TITLE
documentation: Updated documentation

### LINKED ISSUE #139: Update documentation with newly added enhancements
## Description
As a developer,
I need to update the documentation,
so that all the new features are included.

## Mocks
-N.A.-

### Update [Required]
- [ ] Update documentation with information about new features 

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Updated documentation with new enhancements and some small changes

Fixes #139

# Type of Change:
- Code
- Documentation
- This change requires a documentation update (software upgrade on readme file)

# How Has This Been Tested?
-N.A.-

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -163,6 +231,10 @@ hooks on your own repos(for Github), or channels(for Slack), you need
 to change the data accordingly(for Github data change the repo name, 
 owner name etc and for Slack change the channel IDs and user IDs)
 
+In the setup data file, all the data used for github are for the sysbot-test 
+repo which is private. Replace the repo details with your repo if you want to test 
+the data locally
```
Comment (sara-02) [inline docs/README.md]: missing full stop :see_no_evil:

**Comment by ramitsawhney27** [comment]
@sammy1997 Please fix the error mentioned by @sara-02. Also, parse this via Grammarly or some spell checking tool to ensure no grammatical errors are present. @sara-02 can merge this then.

**Comment by sammy1997** [comment]
Updated PR @sara-02 @ramitsawhney27

File: code/main_server.py [hunk 2]
```diff
@@ -31,10 +31,12 @@ def collect_unreviewed_prs():  # pragma: no cover
             message = MESSAGE.get('list_of_unreviewed_prs', '%s') % pr_list[0:-1]
             # Send pr_list to respective channels
             send_message_to_channels(value, message)
+        else:
+            send_message_to_channels(value, MESSAGE.get('no_unreviewed_prs', ""))
 
 
 schedule = BackgroundScheduler(daemon=True)
-schedule.add_job(collect_unreviewed_prs, 'interval', days=7)
+schedule.add_job(collect_unreviewed_prs, 'interval', days=2)
```
Comment (sara-02) [inline code/main_server.py]: Is this a deliberate change?

**Comment by sara-02** [comment]
@sammy1997 Merge conflicts :see_no_evil:

**Comment by sammy1997** [comment]
Resolved @sara-02

**Pull request merged**